### PR TITLE
ci: Fix linter to run on `examples/`

### PR DESCRIPTION
**Pull request by thomass-dev**
Closes https://github.com/probabl-ai/skore/issues/740

### CODE REVIEW

File: examples/plot_01_getting_started.py [hunk 1]
```diff
@@ -17,6 +17,8 @@
 ==========================================================
 """
 
+# ruff: noqa: E402
```
Comment (glemaitre) [inline examples/plot_01_getting_started.py:20]: I would suggest to add the rule in the `pyproject.toml` to not have it in the example.

As an example: https://github.com/scikit-learn/scikit-learn/blob/main/pyproject.toml#L171-L174

Comment (glemaitre) [reply examples/plot_01_getting_started.py:20]: It is less invasive and centralized.

Comment (thomass-dev) [reply examples/plot_01_getting_started.py:20]: Its unfortunately not at the same level (we have chosen a mono-repo/multi-package architecture).
The configuration in `skore/` will not impact `ruff` on the `examples/` directory.
The examples are at the root level, and the python packages in dedicated sub-folders.

I wouldn't like to put examples in the `skore/` directory.

Comment (glemaitre) [reply examples/plot_01_getting_started.py:20]: Arff, in this case, you can have a `ruff.toml` in the root.

Comment (thomass-dev) [reply examples/plot_01_getting_started.py]: I made an in-between: i put the configuration in `examples/ruff.toml` to allow each package to define its own configuration without side-effect.

**Review by glemaitre — APPROVED** [review]
Indeed. It looks like a good trade-off.

**Pull request merged**